Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -308,9 +309,9 @@ namespace pixelTopology {
     static constexpr uint32_t getDoubletsFromHistoMaxBlockSize = 64;  // for both x and y
     static constexpr uint32_t getDoubletsFromHistoMinBlocksPerMP = 16;
 
-    static constexpr uint32_t last_bpix1_detIndex = 108;
-    static constexpr uint32_t last_bpix2_detIndex = 324;
-    static constexpr uint32_t last_barrel_detIndex = 504;
+    static constexpr int16_t last_bpix1_detIndex = 108;
+    static constexpr int16_t last_bpix2_detIndex = 324;
+    static constexpr int16_t last_barrel_detIndex = 504;
```

Comment:
Just curious: why the change from `uint16_t` to `int16_t` ?

Reply:
This should stay `uint16_t`, the compiler was rightfully complaining that because in the soa I had moved the detector index to `int16_t` and that should stay to `uint16_t` too. Summary: moved both  (these and `detectorIndex` in `TrackingRecHitsUtilities` layout back to `uint16_t`.